docs/ql-style-guide.md

```diff
@@ -173,6 +173,7 @@ private predicate foo(Expr e, Expr p) {
 1. Use [camelCase](https://en.wikipedia.org/wiki/Camel_case) for:
    - Predicate names
    - Variable names
+1. Abbreviations and acronyms *should* use normal PascalCase/camelCase (if there are only two letters, it is generally ok to use all uppercase letters).
```

Comment:
This reads poorly, I think, since the meaning gets mixed up if you read past the open parenthesis ("... *should* use normal PascalCase/camelCase if there are only two letters ...")
Possibly change to something like:
```suggestion
1. Abbreviations and acronyms *should* use normal PascalCase/camelCase (as an exception, if there are only two letters then using all uppercase letters is acceptable).
```

Reply:
thanks :+1: